File: test/c/mixed-depth-nesting-3.c
YAY_OBJECT(
    "top", YAY_OBJECT(
        "list", YAY_ARRAY(
            YAY_OBJECT(
                "bytes", yay_bytes_from_hex("b0b5"),
                "tag", yay_string("x")
            ),
            YAY_OBJECT(
                "more", YAY_OBJECT("inner", yay_bytes_from_hex("0f0f"))
            )
        ),
        "solo", yay_int(1)
    )
)

File: test/rs/mixed-depth-nesting-3.rs
Value::Object(HashMap::from([(
    "top".into(),
    Value::Object(HashMap::from([
        (
            "list".into(),
            Value::Array(vec![
                Value::Object(HashMap::from([
                    ("bytes".into(), Value::Bytes(vec![0xb0, 0xb5])),
                    ("tag".into(), Value::String("x".into())),
                ])),
                Value::Object(HashMap::from([(
                    "more".into(),
                    Value::Object(HashMap::from([(
                        "inner".into(),
                        Value::Bytes(vec![0x0f, 0x0f]),
                    )])),
                )])),
            ]),
        ),
        ("solo".into(), Value::Integer(1.into())),
    ])),
)]))
